Sources/Sentry/Public/SentryOptions.h

```diff
@@ -142,6 +142,12 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableAutoSessionTracking;
 
+/**
+ * Whether to attach the top level `operationName` node of HTTP json requests to HTTP breadcrumbs
+ * @note Default is @c NO.
+ */
+@property (nonatomic, assign) BOOL enableGraphQLOperationTracking;
```

Comment:
`m`: Please add some tests for this new property. All you have to do is to add a similar test as this one https://github.com/getsentry/sentry-cocoa/blob/32ac9341b30033766e7ff03cd2e29592ad90e6ff/Tests/SentryTests/SentryOptionsTest.m#L198-L201